Gopkg.toml

```diff
@@ -84,7 +84,7 @@
   name = "github.com/banzaicloud/bank-vaults"
 
 [[constraint]]
-  branch = "master"
+  branch = "update-creation"
```

Comment:
I just merged this branch to master https://github.com/banzaicloud/banzai-types/pull/42#pullrequestreview-113099298

Reply:
Updated to master